lib/signaling/v2/room.js

```diff
@@ -220,6 +229,20 @@ class RoomV2 extends RoomSignaling {
     return this;
   }
 
+  /**
+   * @private
+   */
+  _createRemoteParticipant(participantState) {
+    const { _RemoteParticipantSignaling: RemoteParticipantV2 } = this;
+    return new RemoteParticipantV2(
+      participantState,
+      trackSid => this._getInitialTrackSwitchOffState(trackSid),
```

Comment:
Tracks are initially on in v2. If they've since been switched off when the RemoteParticipant is created (MSP beats RSP) will this logic suppress the initial off event? I don't think its desirable to do this, but I think it was pre-existing behavior.

Reply:
@ceaglest I don't think this will suppress the initial switch off event, since the SFU will not turn it off until the SDK detects that there are no attached visible video elements in the subscription event handler, and sends an off request.
The only time what you are saying may happen is if the SFU switches off the track due to congestion just before the track is created.